nuxt.config.ts

```diff
@@ -270,6 +270,8 @@ const config: NuxtConfig = {
         server: '#424242'
       }
     },
+    optimizeCSS: true,
+    extractCSS: true,
```

Comment:
So do we keep this?

Reply:
@ThibaultNocchi I'm not having any issues building locally. Have you tried in an perfectly clean environment? I wonder if you're getting conflicts with cache artifacts.